specification/ml/_types/TrainedModel.ts

```diff
@@ -121,6 +123,11 @@ export class TrainedModelSizeStats {
 export class TrainedModelDeploymentNodesStats {
   /** The average time for each inference call to complete on this node. */
   average_inference_time_ms: double
+  /**
+   * The average time for each inference call to complete on this node
+   * in the last minute. @Since 8.2.0
+   */
+  average_inference_time_ms_last_minute: double
```

Comment:
[From the PR](https://github.com/elastic/elasticsearch/pull/84628/files#diff-ef1e72f9ddf1f055b78b94a380131fe4d3c81a7e1926c9801616a25078a87011R272) this field looks optional?